src/azure-cli/azure/cli/command_modules/resource/_validators.py

```diff
@@ -8,6 +8,8 @@
 import argparse
 from azure.cli.core.azclierror import ArgumentUsageError
 
+from azure.cli.core.azclierror import InvalidArgumentValueError
+from azure.cli.core.commands import parameters
```

Comment:
If the models are less used, it is better to import them close to where they are used.